api/src/test/java/io/opentelemetry/common/AttributesTest.java

```diff
@@ -149,6 +150,8 @@ void builder() {
             "value1",
             longKey("long"),
             100L,
+            longKey("long2"),
+            10L,
```

Comment:
hm. a shame it doesn't fix this usage as well. :(

Reply:
I did run into 2 of these in java-instrumentation, but just realized they probably should have been long in those particular cases anyways:

https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1266/commits/a83da9ed4fd0a26294391b45b34fc1404a2b4421